src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -188,7 +188,6 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
     <ApplicationView
       onUpdate={onUpdate}
       onSave={onSave}
-      initialValue={currentResponse}
```

Comment:
You can see here that the existing `initalValue` prop is just the same thing as `currentResponse`, so we can get rid of it.